src/components/Settings.jsx

```diff
@@ -86,6 +93,54 @@ export default function Settings({ currentWallet }) {
           />
           Use {settings.useAdvancedWalletMode ? 'magic' : 'advanced'} wallet mode
         </rb.Button>
+
+        <br />
+
+        <rb.Button
+          variant="outline-dark"
+          className="border-0 mb-2 d-inline-flex align-items-center"
+          onClick={async (e) => {
+            e.preventDefault()
+            setError(false)
+            setRevealSensitiveInfo(false)
+            if (!showingSeed) {
+              const { name: walletName, token } = currentWallet
+              const res = await Api.getSeed({ walletName, token })
+              if (res.ok) {
+                const { seedphrase } = await res.json()
+                setSeed(seedphrase)
+                setShowingSeed(!showingSeed)
+              } else {
+                setError(true)
+              }
+            } else {
+              setShowingSeed(!showingSeed)
+            }
+          }}
+        >
+          <Sprite symbol="mnemonic" width="24" height="24" className="me-2" />
+          {showingSeed ? 'Hide' : 'Show'} seed phrase
+        </rb.Button>
+        {error && (
```

Comment:
```suggestion
        {showingSeed && error && (
```

Let's only show the error in this case.

Reply:
I dont think this will work because we only set `showingSeed` to be true if the API call works, if it fails then we do not set it to true because there is no seed to display and we show the error instead.